### ISSUE TITLE
[Bug Report][2.5.8] Incorrect 'role' attribute value present on v-dialog component

### ISSUE THREAD

**Issue by phylot** [vuetifyjs/vuetify#14231]
### Environment
**Vuetify Version:** 2.5.8
**Vue Version:** 2.6.11
**Browsers:** Chrome 94.0.4606.61
**OS:** Mac OS 10.15.7

### Steps to reproduce
Create a v-dialog component and make it visible.

### Expected Behavior
The 'role' attribute applied to the dialog should be role="dialog".

### Actual Behavior
The role applied to the dialog is role="document".

### Reproduction Link
<a href="https://codepen.io/zerobob/pen/WNOBWwr" target="_blank">https://codepen.io/zerobob/pen/WNOBWwr</a>
